cbits/blake2/sse/blake2s.c

```diff
@@ -290,7 +290,7 @@ int blake2s( void *out, size_t outlen, const void *in, size_t inlen, const void
 #if defined(SUPERCOP)
 int crypto_hash( unsigned char *out, unsigned char *in, unsigned long long inlen )
```

Comment:
Is this inteded to be `crypto_` or is a typo?

Reply:
It's that way in the [official source](https://github.com/BLAKE2/BLAKE2/blob/master/sse/blake2s.c#L291), so I assume it's designed to interface with the [`SUPERCOP` benchmark suite](https://bench.cr.yp.to/supercop.html).